core/state/database.go

```diff
@@ -64,37 +64,37 @@ type Database interface {
 	// TrieDB retrieves the low level trie database used for data storage.
 	TrieDB() *trie.Database
 
-	StartVerkleTransition(originalRoot, translatedRoot common.Hash, chainConfig *params.ChainConfig, cancunTime *uint64)
+	StartVerkleTransition(originalRoot common.Hash, chainConfig *params.ChainConfig, cancunTime *uint64, root common.Hash)
```

Comment:
TODO: see if `originalRoot` and `root` can be merged into a single value. It should be possible as the verkle transition is only started at a given block height, which is the one at the fork.